hedera-mirror-importer/src/main/resources/db/scripts/addMissingTokenAccountAssociation.sql

```diff
@@ -0,0 +1,28 @@
+-------------------
+-- Add the missing token account association
+-------------------
+
+-- Importer has missed the following auto token-account associations:
+--   1. A token's treasury at the time of token creation
+--   2. A token's custom fee collector of either a fixed custom fee charged in the newly created token or a fractional
+--      fee, at the time of token creation
+-- Once the importer is patched, we can add associations for the token-account pairs in the account balance file but
+-- not in the token_account table
+insert into
+    token_account (account_id, associated, created_timestamp, freeze_status, kyc_status, modified_timestamp, token_id)
+select
+    tb.account_id,
+    true,
+    t.created_timestamp,
+    case when t.freeze_key is null then 0
+         else 2
```

Comment:
Need to consider default freeze
```suggestion
         when token.freeze_default =  true then 1 else 2
```

Reply:
Ignore based on special case for treasury